### DISCUSSION TITLE
css files getting 404 errors

### DISCUSSION THREAD

**Discussion by zunder1990** [pretalx/pretalx#1657] [Self-hosting] [answered]
Moving this here as this probably a better place for this.
We made our CFP open today and now we are getting 404 errors on the css files
https://speakers.southeastlinuxfest.org/southeast-linux-fest-2024/
I ran an update steps just to make sure including regen css files.
https://docs.pretalx.org/administrator/maintenance.html

This only happens for unauthorized users, event admins when logged in see everything fine.

All CSS is set to default.
any points would be nice.

the two files that I am getting 404 errors are found /var/pretalx/data/media/southeast-linux-fest-2024 and owned by the pretalx user.
I tired to hard set media = /var/pretalx/data/media/ in the pretalx.cfg file but made no difference 

version 2023.1.3 running on debian 11.7
https://github.com/pretalx/pretalx/issues/1655

**Comment by rixx** [comment]
I can only reiterate my statement over at the issue: the most likely cause is that your web server isn't configured correctly. Steps you should take:

1. Verify that the exact file that fails to load (e.g. `media/southeast-linux-fest-2024/cfp.140b59d7ab70017c.css`) exists in your file system
2. Verify that your web server can access this file
3. Double-check this by picking any other image in your `media` directory, e.g. a user avatar, and check if you can access this file via HTTP
4. Check your web server's error logs – if you're running NGINX, you can also check that your config matches our documentation.

**Comment by zunder1990** [comment, marked as answer]
Ok I have gotten to the bottom of this and it is now fixed. Before I had haproxy doing the ssl but it was missing from folder config. I removed haproxy and followed the guild to get NGINX installed and running.
